homeassistant/components/freebox/router.py

```diff
@@ -146,6 +153,32 @@ async def _update_disks_sensors(self) -> None:
         for fbx_disk in fbx_disks:
             self.disks[fbx_disk["id"]] = fbx_disk
 
+    async def update_home_devices(self) -> None:
+        """Update Home devices (light, cover, alarm, sensors ...)."""
+        new_device = False
+        # Home sensors (alarm, pir, switch, remote ...)
+        if self.home_granted is True:
+            try:
+                home_nodes: list[Any] = await self._api.home.get_home_nodes() or []
+            except InsufficientPermissionsError:
+                self.home_granted = False
+                _LOGGER.warning("Home access is not granted")
+                return
+        else:
+            return
```

Comment:
```suggestion
        if not self.home_granted:
            return

        new_device = False
        # Home sensors (alarm, pir, switch, remote ...)
        try:
            home_nodes: list[Any] = await self._api.home.get_home_nodes() or []
        except InsufficientPermissionsError:
            self.home_granted = False
            _LOGGER.warning("Home access is not granted")
            return

```
To avoid adding indentation and reading complexity